tools/make/tests.toml

```diff
@@ -45,11 +45,10 @@ set_env CARGO_TARGET_DIR ${pwd}/target
 cd docs/tutorials/testing
 
 exec --fail-on-error cargo --config patch.toml test -p tutorials-test
-exec --fail-on-error cargo --config patch.toml check -p tutorial_baked
+exec --fail-on-error cargo --config patch.toml check -p tutorial_default
 exec --fail-on-error cargo --config patch.toml check -p tutorial_buffer
 exec --fail-on-error cargo --config patch.toml check -p tutorial_experimental
 exec --fail-on-error cargo --config patch.toml check -p tutorial_lazy_static
-exec --fail-on-error cargo --config patch.toml check -p tutorial_testdata
```

Comment:
custom_compiled?

Reply:
I guess we could build it without custom data, but what's the point. This check isn't used anyway.

Reply:
ok. Can you leave a comment explaining why you're intentionally omitting custom_compiled from this list?